### DISCUSSION TITLE
Naxx 25 Wotlk Boost

### DISCUSSION THREAD

**Discussion by PerturaboS** [chromiecraft/chromiecraft#7117] [Suggestions]
Hello,

As we are closer and closer to 80 lvl content, I would like to suggest some changes in Naxx 25 to make it at least challange, not like Blizzlike - clown fiesta run with blind eyes.

We all know, that server on current Player base is more a like handicap PvE guild fights... Mytic was too hard for half server, Magtheridon was only for 1 maybe 2 guilds... but hear me out.

Anub’Rekhan:
+Immune to Taunt
+Every 20-30s (depends on tests) boss shouts "More more more!" and 3s after that:
-clean Agro Table
-summon 2 Crypt Guard (from his starting point gate, on his back)
+Impale as it is in Blizzard version.

Once he goes in to Locust phase on fixed HP parts (75%-50%-25%) he gains movement speed - 110% and deal 500% melee dmg.
This phase takes aproximently  20-30s (to be decided if we want to include 1 Agro table wipe - A bit stress for Kiter is welcome)

And in the end, we should bring 10% Enrage which will force proxing some defence/reduction dmg abilities and possible tank swap.

Summing Up: Its still same Anub, but requires more pay attention by players and team co-operation.

What you think about it?

Thanks,
Pert

**Comment by heyitsbench** [comment]
I personally despise any idea of modified mechanics period. This server aims for Blizzlike, specifically to fix bugs and issues in that goal. If we make custom mechanics, first and foremost any issues the core has in terms of Blizzlike behavior will not be found by players. This is also part of why I dislike custom tuning for bosses at all.

Staff have also said in the past they will not be doing any custom tuning on Wrath raid/dungeon content, I highly doubt they'd change up mechanics given that.

**Comment by PerturaboS** [comment]
We are 16 years after WOTLK content and I guess everyone already played it 5-8 times... Bringing same raids, without any changes will not bring news players to server, as currently every year we can see 3-4 fresh Wotlk servers which are Blizzlike. In the past, i played on modified content, where bosses where tuned and with additional mechanics. Server had stable 2k population, even full max clear was possibile oby for 1 guild. Why ? Because IT was something new and challanging... Not the same...

**Reply by heyitsbench** [reply]
Then you simply aren't familiar with the goal of the server.
